src/Tasks.UnitTests/RemoveDuplicates_Tests.cs

```diff
@@ -19,14 +16,16 @@ sealed public class RemoveDuplicates_Tests
         [Fact]
         public void OneItemNop()
         {
-            RemoveDuplicates t = new RemoveDuplicates();
+            var t = new RemoveDuplicates();
             t.BuildEngine = new MockEngine();
 
             t.Inputs = new ITaskItem[] { new TaskItem("MyFile.txt") };
 
             bool success = t.Execute();
             Assert.True(success);
+            Assert.Equal(1, t.Filtered.Length);
             Assert.Equal("MyFile.txt", t.Filtered[0].ItemSpec);
+            Assert.False(t.HadAnyDuplicates);
```

Comment:
I augmented these C# tests, but couldn't find any .proj tests for RemoveDuplicates (e.g., to make sure the `[Output]` attribute is working correctly. (I did some manual testing here: it's working.) Are these unit tests enough?